### PR TITLE
add caveat for network plugins in swarm mode

### PR DESCRIPTION
**Pull request by sfsmithcha**
added caveat re: swarm mode to network plugins doc

Signed-off-by: Charles Smith charles.smith@docker.com

### CODE REVIEW

**Comment by sfsmithcha** [comment]
ping @mgoelzer @mavenugo

File: docs/extend/plugins_network.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ LibNetwork, which shares plugin infrastructure with Engine. Effectively, network
 driver plugins are activated in the same way as other plugins, and use the same
 kind of protocol.
 
+## Network driver plugins and swarm mode
+
+Docker 1.12 adds support for cluster management an orchestration called
```
Comment (thaJeztah) [inline docs/extend/plugins_network.md]: s/an/and/ ?

File: docs/extend/plugins_network.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ LibNetwork, which shares plugin infrastructure with Engine. Effectively, network
...
+[swarm mode](../swarm/index.md). Docker Engine running in swarm mode currently
+only supports the built-in overlay driver for networking. Therefore existing
+networking plugins will not work in swarm mode.
+
+When you run Docker Engine outside of swarm mode, all networking plugins that
+worked in Docker 1.11 will continue to function normally.  They do not require
```
Comment (thaJeztah) [inline docs/extend/plugins_network.md]: nit: there's two spaces before "They"

**Comment by thaJeztah** [comment]
couple of nits, but LGTM otherwise

**Comment by mgoelzer** [comment]
LGTM

**Pull request merged**